catalogue_pipeline/ingestor/src/main/scala/uk/ac/wellcome/platform/ingestor/services/IngestorWorkerService.scala

```diff
@@ -20,11 +20,13 @@ class IngestorWorkerService @Inject()(
   metrics: MetricsSender
 ) extends SQSWorker(reader, system, metrics) {
 
-  override def processMessage(message: SQSMessage): Future[Unit] = Future {
+  override def processMessage(message: SQSMessage): Future[Unit] =
     fromJson[Work](message.body) match {
       case Success(work) =>
         identifiedWorkIndexer.indexWork(work = work).map(_ => ())
-      case Failure(err) => throw GracefulFailureException(err)
+      case Failure(err) =>
+        Future {
```

Comment:
You should do `Future.failed(new GracefulFailureException(err))`

Reply:
Actually you could just do 
```
for{
work <- Future.fromTry(fromJson[Work](message.body))
_ <- identifiedWorkIndexer.indexWork(work = work).map(_ => ())
} yield ()
```
`fromJson` is already mapping any exception to `GracefulFailureException` so you don't need to to that again